src/Library/demos/Spin Button/main.blp

```diff
@@ -0,0 +1,69 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Spin Button";
+  description: _("Let users choose a precise numerical value");
+
+  Box {
+    orientation: vertical;
+    spacing: 12;
+    Label {
+      label: "Select Time";
+    }
```

Comment:
I think this label should have one of the CSS title classes, or maybe the parent box should have the `card` CSS class, to separate it from the labels above and below.